configs/workbench.json

```diff
@@ -1,11 +1,38 @@
 {
   "index_name": "workbench",
-  "start_urls": ["https://workbench.gusto.com/"],
+  "start_urls": [
+    {
+      "url": "https://workbench.gusto.com/"
+    },
+    {
+      "url": "https://workbench.gusto.com/getting-started/",
+      "page_rank": 1
+    },
+    {
+      "url": "https://workbench.gusto.com/foundations/",
+      "page_rank": 2
+    },
+    {
+      "url": "https://workbench.gusto.com/components/",
+      "page_rank": 3
+    },
+    {
+      "url": "https://workbench.gusto.com/patterns/"
+    },
+    {
+      "url": "https://workbench.gusto.com/content/"
+    },
+    {
+      "url": "https://workbench.gusto.com/resources/"
+    },
+    {
+      "url": "https://workbench.gusto.com/updates/"
+    }
+  ],
   "stop_urls": ["\\/\\?"],
-  "sitemap_urls": ["https://workbench.gusto.com/sitemap/sitemap-0.xml"],
   "selectors": {
     "lvl0": {
-      "selector": ".DocSearch-active-menu-item",
+      "selector": ".DocSearch-l0",
```

Comment:
Selector seems valid, is there any client-side rendered content?

Reply:
The previous selector (`.DocSearch-active-menu-item`) was part of our sidenav, which is partially rendered in a React `useEffect` so maybe there was a delay and I needed to add a `js_wait`. Unfortunately I cannot test it, so I would be making a random guess. 

The new selector (`.DocSearch-l0`) is a hidden `span` on each page with a string specifying its L0 value. It is rendered the same as all the other text content on the page, so I hope that it will extract properly.

Reply:
Perfect!

The new infra will allow testing crawls etc. so keep an eye on your emails!